airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -265,8 +305,8 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
 
     def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         authenticator = TokenAuthenticator(config["api_token"])
-        default_start_date = pendulum.now().subtract(days=14)  # TODO make this configurable
-        threads_lookback_window = {"days": 7}  # TODO make this configurable
+        default_start_date = datetime.strptime(config["start_date"], "%Y-%m-%dT%H:%M:%SZ")
+        threads_lookback_window = {"days": config["lookback_window"]}
```

Comment:
pass Interval instead, don't need to abuse dict, WDYT?

Reply:
@keu like from date to date? I think it is not what was required

Reply:
```suggestion
        threads_lookback_window = pendulum.Interval(days=config["lookback_window"])
```

Reply:
Done